### ISSUE TITLE
Extend ThymeLeaf usage by Dialect

### ISSUE THREAD

**Issue by pitpalme** [jbake-org/jbake#195]
Hello,

in jbake-example-project-thymeleaf/templates/header.thyme there's an IE conditional include of javascript.
Unluckily it's not rootpath relative, but statically absolute to "/".

I'd like to extend "ThymeleafTemplateEngine" to enable usage of "ConditionalCommentsDialect" - if present on ClassPath.

What's the general opinion about this? Any reasons this shouldn't be there?

I'd open a PR if it's OK to introduce this ThymeLeaf dialects usage ...

Regard,
Peter

**Comment by jonbullock** [comment]
Hi Peter,

Is it possible to use content.rootpath value for the IE conditional to make the path relative?

Jon

**Comment by pitpalme** [comment]
Hi Jon,

yes it is. I've extended "header.thyme" to make use of "content.rootpath" for "js/html5shiv.min.js", but the surrounding 

```html
    <!--[if lt IE 9]>
    ...
    <![endif]-->
```

prevents ThymeLeaf from evaluating the dynamic part.
I've managed to write some ThymeLeaf code for writing the comments as part of generated output, instead of template, but that's ugly. Than I discovered mentioned ThymeLeaf extension, addressing exactly this issue and thought it might me much more elegant and less error prone.

Peter
